kustomize/internal/commands/build/flagaddmanagedby.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package build
+
+import (
+	"github.com/spf13/pflag"
+)
+
+const (
+	flagEnableManagedbyLabelName = "enable_managedby_label"
+	flagEnableManagedbyLabelHelp = `enable adding a managed-by label to the kustomize output
+	app.kuberentes.io/managed-by: kustomize-<version>
```

Comment:
spelling.

actually, can you include the actual constant here to avoid this, e.g.

```
help = `enable adding ' + konfig.ManagedByLabelKey
```
or something.  likely have to move the constant (that's ok)

Reply:
Done. Moved the  `ManagedByLabelKey` from `krusty` to `konfig`.